smpte.js

```diff
@@ -632,44 +637,80 @@ function insertConformance(docMetadata) {
+    is preferred but not necessarily required; or that (in the negative form) a certain possibility
+    or course of action is deprecated but not prohibited.</p>
+
+  <p>The keywords "may" and "need not" indicate courses of action permissible within the limits of the document. </p>
+
+  <p>The keyword "reserved" indicates a provision that is not defined at this time, shall not be used,
+    and may be defined in the future. The keyword "forbidden" indicates "reserved" and in addition
+    indicates that the provision will never be defined in the future.</p>
 
-  <p>All text in this document is, by default, normative, except: the Introduction, any clause explicitly
-  labeled as "Informative" or individual paragraphs that start with "Note:" </p>
+  ${implConformance}
 
-<p>The keywords "shall" and "shall not" indicate requirements strictly to be followed in order to conform to the
-document and from which no deviation is permitted.</p>
+  <p>Unless otherwise specified, the order of precedence of the types of normative information in
+    this document shall be as follows: Normative prose shall be the authoritative definition;
+    tables shall be next; then formal languages; then figures; and then any other language forms.</p>
+    `;
 
-<p>The keywords, "should" and "should not" indicate that, among several possibilities, one is recommended
-  as particularly suitable, without mentioning or excluding others; or that a certain course of action
-  is preferred but not necessarily required; or that (in the negative form) a certain possibility
-   or course of action is deprecated but not prohibited.</p>
+  } else {
+
+    sec.innerHTML = `
+    <h2>Conformance</h2>
+    <p>This Engineering Guideline is purely informative and meant to provide tutorial information to the 
+    industry. It does not impose Conformance Requirements and avoids the use of Conformance Notation.</p>
 
-<p>The keywords "may" and "need not" indicate courses of action permissible within the limits of the document. </p>
+    <p>Engineering Guidelines frequently provide tutorial information about a Standard or Recommended Practice
+    and when this is the case, the user should rely on the Standards and Recommended Practices referenced for
+    interoperability information.</p>
+    `;
 
-<p>The keyword "reserved" indicates a provision that is not defined at this time, shall not be used,
-  and may be defined in the future. The keyword "forbidden" indicates "reserved" and in addition
-   indicates that the provision will never be defined in the future.</p>
+  }
 
-${implConformance}
+  if (docMetadata.pubType === smpte.EG_PUBTYPE) {
+    
+    const sections = document.querySelectorAll('section');
+    sections.forEach((element) => {
+      let id = element.id;
+      if ((id !== "sec-front-matter") && (id !== "sec-foreword") && (id !== "sec-conformance")) {
+        if (element.innerText.includes("shall")) {
```

Comment:
We need to handle uppercase and lowercase.

... and what about equivalent forms, e.g., "need to be"? See Clause 7 at https://www.iso.org/sites/directives/current/part2/index.xhtml

Reply:
> We need to handle uppercase and lowercase.

Addressed in https://github.com/SMPTE/html-pub/pull/287/commits/c17ea952c02ab852df768568fad1223874fec25b
 
> ... and what about equivalent forms, e.g., "need to be"? See Clause 7 at https://www.iso.org/sites/directives/current/part2/index.xhtml

I'm using this text from the `Conformance` clause we use:

```
Normative text is text that describes elements of the design that are indispensable or contains the conformance language keywords: "shall", "should", or "may". Informative text is text that is potentially helpful to the user, but not indispensable, and can be removed, changed, or added editorially without affecting interoperability. Informative text does not contain any conformance keywords
```

I would argue that the keywords are what are usually used, and need to be flagged. Past that it's an editorial thing, and I cannot recall ever seeing any of those in a SMPTE doc TBH. Also, do EGs docs NEED to conform to ISO guidelines for equivalent forms? I'm just using the OM as a baseline as a first pass.